sub-packages/bionemo-testing/src/bionemo/testing/data/resources/single_cell.yaml

```diff
@@ -21,3 +21,9 @@
   sha256: ab038b184de52e53ff7bcea5e01d97d55944c507db88c0495bdf9e5e9e0303a4 # pragma: allowlist secret
   owner: John St John <jstjohn@nvidia.com>
   description: Golden values for geneformer QA model.
+
+- tag: testdata-memmap-format
+  pbss: "s3://bionemo-ci/test-data/singlecell/ordered_singlecell_mmap_test_data.tar.gz"
+  sha256: d1d3da1c831877100e55b326e4588ffe305638a06c77415468344bd63ea3657c # pragma: allowlist secret
+  owner:  Savitha Srinivasan <savithas@nvidia.com>
+  description: Test data for single cell models in SCDL Memmap format.
```

Comment:
Is there a plan to upload this to NGC? Not sure how this works now that we're public, any tests that use this would fail for non-NV folks

Reply:
Yes! I'm working with Dorota on this.